https://bugs.gentoo.org/899770
https://github.com/MythTV/mythtv/commit/3b76df2bb45789f075b6b83d6e6b436aebaf2e83

From 3b76df2bb45789f075b6b83d6e6b436aebaf2e83 Mon Sep 17 00:00:00 2001
From: David Hampton <mythtv@love2code.net>
Date: Thu, 19 Jan 2023 14:52:28 -0500
Subject: [PATCH] Assume existence of stdint.h in exiv2.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Based on this commit to exiv2.

    Author: Luis Díaz Más <piponazo@gmail.com>
    Date:   Wed Sep 16 21:40:05 2020 +0200

        Assume existence of stdint.h
--- a/external/libexiv2/xmpsdk/include/MD5.h
+++ b/external/libexiv2/xmpsdk/include/MD5.h
@@ -25,18 +25,7 @@
 
 #include <sys/types.h>
 
-#ifdef _MSC_VER
-//   _MSC_VER   1600 == Visual Studio 2010
-# if _MSC_VER < 1600
-#  ifdef  EXV_HAVE_STDINT_H
-#   undef EXV_HAVE_STDINT_H
-#  endif
-# endif
-#endif
-
-#if defined(EXV_HAVE_STDINT_H) || defined(__MINGW32__) || defined(__MING64__) || defined(__APPLE__)
-# include <stdint.h>
-#endif
+#include <cstdint>
 
 /* MSVC doesn't provide C99 types, but it has MS specific variants */
 #ifdef _MSC_VER

